CMakeLists.txt

```diff
@@ -423,6 +423,20 @@ if(APPLE AND NOT (${CMAKE_VERSION} VERSION_LESS 2.8.12))
     endif()
 endif()
 
+set(OPENSIM_DEPENDENCIES_DIR 
+    "" 
+    CACHE 
+    PATH
+    "Directory containing installed binaries of OpenSim dependencies.")
```

Comment:
Can you clarify that this related to the superbuild and should not be set if you installed the dependencies yourself? People might get confused if they built their dependencies themselves; they might think they ALSO have to set this.